<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadWareHouse</class>
 <widget class="QWidget" name="UploadWareHouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UploadWareHouse</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中商品编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>商品数量：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countEdit">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="showLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="uploadBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>商品入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloaseBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出操作</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
